src/UCTSearch.cpp

```diff
@@ -325,6 +324,7 @@ Move UCTSearch::think(BoardHistory&& new_bh) {
         dump_analysis(milliseconds_elapsed, true);
     }
     Move bestmove = get_best_move();
+    myprintf("selected %s [%s]\n", UCI::to_string(bestmove).c_str(), bh_.cur().move_to_san(bestmove).c_str());
```

Comment:
I think this is redundant with UCI.cpp which also outputs bestmove? I see you included move_to_san, if that's the goal maybe move this to UCI.cpp instead?